mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/Analytics.java

```diff
@@ -204,21 +205,18 @@ public static void persistResource(MindmapsGraph mindmapsGraph, Vertex vertex,
         RoleType resourceValue = transaction.getRoleType(GraqlType.HAS_RESOURCE_VALUE.getId(resourceName));
         RelationType relationType = transaction.getRelationType(GraqlType.HAS_RESOURCE.getId(resourceName));
 
-        Instance instance = transaction.getInstance(vertex.value(DataType.ConceptPropertyUnique.ITEM_IDENTIFIER.name()));
+        Instance instance =
```

Comment:
We compute the degrees for almost all the entities, and then persist a degree for each of the vertices.